code/gallery.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Gallery</title>
+    <link href="style.css" rel = "stylesheet" type="text/css"> 
+
+  </head>
+<body>
+  <h1>Gallery!</h1>
+    <div style="text-align: center"><img src="/Users/livartda/Documents/project-p1-first-webpage-master/code/SAM_1220.JPG" width="800" height="550"></div>
+    <menu>
+        <a href="/Users/livartda/Documents/project-p1-first-webpage-master/code/index.html">Home</a>
+        <a href="/Users/livartda/Documents/project-p1-first-webpage-master/gallery.html">Gallery</a>
+      </menu>
+      
+      <p class="box-one">
+As a person that likes to travel, I often bring my camera with me to snap some photos of the places I go to.
+<br>
+<br>Below is a few of the photos I have captured during the years while travelling to different places in Europe, but also my home country Norway
+
+<br>If you click on the photos below they will open in a new tab in your browser
+    </p>
+
+
+   <div class="box-one">   
+<div class="gallery">
+  <a target="_blank" href="/Users/livartda/Documents/project-p1-first-webpage-master/code/IMG_2282-1.jpg">
+    <img src="/Users/livartda/Documents/project-p1-first-webpage-master/code/IMG_2282-1.jpg" alt="City life">
+  </a>
+  <div class="desc">Oslo city during an autumn evening</div>
+  
+</div>
+
+<div class="gallery">
+  <a target="_blank" href="/Users/livartda/Documents/project-p1-first-webpage-master/code/WP_20150124_018.jpg">
+    <img src="/Users/livartda/Documents/project-p1-first-webpage-master/code/WP_20150124_018.jpg" alt="Forest">
+  </a>
+  <div class="desc">Svalbard in the month of January where it is dark all day and night</div>
+</div>
+
+<div class="gallery">
+  <a target="_blank" href="/Users/livartda/Documents/project-p1-first-webpage-master/code/IMG_2183.JPG">
+    <img src="/Users/livartda/Documents/project-p1-first-webpage-master/code/IMG_2183.JPG" alt="Northern Lights">
+  </a>
+  <div class="desc">Cardiff Castle, Wales</div>
+</div>
+
+<div class="gallery">
+  <a target="_blank" href="/Users/livartda/Documents/project-p1-first-webpage-master/code/DSC_0054.JPG">
+    <img src="/Users/livartda/Documents/project-p1-first-webpage-master/code/DSC_0054.JPG" alt="Mountains">
+  </a>
+  <div class="desc">City of Budapest, Hungary</div>
+</div>
+
+<div class="gallery">
+  <a target="_blank" href="/Users/livartda/Documents/project-p1-first-webpage-master/code/IMG_0020R.png">
+    <img src="/Users/livartda/Documents/project-p1-first-webpage-master/code/IMG_0020R.png" alt="Mountains">
+  </a>
+  <div class="desc">Trondheim, Norway</div>
+</div>
+
+<div class="gallery">
+  <a target="_blank" href="/Users/livartda/Documents/project-p1-first-webpage-master/code/SAM_1411.JPG">
+    <img src="/Users/livartda/Documents/project-p1-first-webpage-master/code/SAM_1411.JPG" alt="Mountains">
```

Comment:
I miss Norway! One of the best hikes I've ever been on was in Åndalsnes, Romsdalseggen. I had to google Molladalen now, it looks spectacular 😍

Reply:
Romsdalseggen is a really nice place to hike, but Molladalen is breathtaking!